snmp/datadog_checks/snmp/data/default_profiles/silverpeak-edgeconnect.yaml

```diff
@@ -0,0 +1,102 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "silverpeak"
+
+sysobjectid: 1.3.6.1.4.1.23867.1.2.*  # Silverpeak devices
+metrics:
+#  - MIB: SILVERPEAK-MGMT-MIB
+#    symbol:
+#      OID: 1.3.6.1.4.1.23867.3.1.1.1.1.0
+#      name: silverpeak.mgmt.spsSystemVersion
+# scalar string metric is not supported yet (keep this metric and this comment in profile until it's fixed)
+#  - MIB: SILVERPEAK-MGMT-MIB
+#    symbol:
+#      OID: 1.3.6.1.4.1.23867.3.1.1.1.2.0
+#      name: silverpeak.mgmt.spsProductModel
+# scalar string metric is not supported yet (keep this metric and this comment in profile until it's fixed)
+#  - MIB: SILVERPEAK-MGMT-MIB
+#    symbol:
+#      OID: 1.3.6.1.4.1.23867.3.1.1.1.3.0
+#      name: silverpeak.mgmt.spsOperStatus
+# scalar string metric is not supported yet (keep this metric and this comment in profile until it's fixed)
```

Comment:
from the descriptions in the MIB, these look like they can be added as metadata

Reply:
But [these](https://github.com/DataDog/snmp-mibs/blob/0b6c2eeec694d9e1e16aab0c8022f646dd2b01c3/mibs/SILVERPEAK-MGMT-MIB#L48-L58) are scalar string metrics ? When I had them, the assertions were failing cause they couldn't find the metrics, although they were present in snmprec